qiskit/wrapper/jupyter/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=import-error
```

Comment:
This one seems legit :tada:. However, can you move it to the lines where the jupyter/ipython imports are invoked, to ensure the scope of the disable is limited?

```
from IPython import get_ipython  # pylint: disable=import-error
from .jupyter_magics import StatusMagic
```